addons/project/models/project_task.py

```diff
@@ -91,7 +91,8 @@ def _get_default_stage_id(self):
 
     @api.model
     def _default_personal_stage_type_id(self):
-        return self.env['project.task.type'].search([('user_id', '=', self.env.user.id)], limit=1).id
+        default_id = self.env.context.get('default_personal_stage_type_ids')
```

Comment:
Imo this is a general improvement for the model project.task personal stage system. The change is needed because we introduce the editable list view on that model in Todo but it can be helpful if we decide to do that in project or other module using this model in the future.

Reply:
Okay, indeed, you can let as you did, thanks for the explanation. 🙂